src/asset/hollowknight/categories/white-defender.json

```
@@ -0,0 +1,22 @@
+{
+    "categoryName": "White Defender%",
+    "splitIds": [
+        "VengefulSpirit",
+        "EnterGreenpath",
+        "MothwingCloak",
+        "MantisClaw",
+        "BlueLake",
+        "DreamNail",
+        "GorgeousHusk",
+        "EnterSanctum",
+        "DesolateDive",
+        "TollBenchCity",
+        "SlySimpleKey",
+        "DungDefender",
+        "Godhome",
```

Brand new today, we have Godhome (Transition). Let's use that instead. Either you'll have pull latest main into this branch, or redo the PR off latest main
```suggestion
        "EnterGodhome",
```